### DISCUSSION TITLE
Make icons circular

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2124] [Ideas]
There should be an option in the 2.0-3.0 addon to make icons circular.

**Comment by mxmou** [comment]
Which icons?

**Reply by ItsEnderium** [reply]
User icons.

**Reply by ItsEnderium** [reply]
Like in GitHub

**Reply by mxmou** [reply]
Then this is unrelated to 2.0 → 3.0 because in the 3.0 design icons are not circular. It could be a separate addon but I don't think it would be very useful.
